lib/ledger.js

```diff
@@ -84,7 +84,9 @@ async function _findAgent() {
 async function _setupGenesisNode() {
   const ledgerOwner = await _getLedgerOwner();
   const ledgerConfiguration = cfg.config;
-
+  if(!ledgerConfiguration) {
+    throw new BedrockError('Genesis node configuration required.', 'DataError');
```

Comment:
I like the error handling.  Can the error message be more to the point?

`Ledger configuration not found.  "bedrock.config['ledger-core'].config" not set.`, 'NotFoundError'

I don't love the idea of being so explicit here giving the exact config, but really, isn't that what is needed to address the issue.  Otherwise, one must follow the stack trace etc.  

It is my feeling generally that `DataError` is an error of last resort, I think `NotFoundError` or `InvalidStateError` is more appropriate here.

Reply:
https://github.com/digitalbazaar/bedrock-ledger-core/pull/4/commits/b1c0d9ffabd59cefca86f0e472875c49969b844c

p.s. I am retesting this with every change made so if possible let's not be to nit picky here and it is delaying work on another project.

Reply:
OK, I'll let someone else comment on the style issue: https://github.com/digitalbazaar/eslint-config-digitalbazaar/issues/30

Reply:
I can fix that on Friday if I get time.